src/python/AsyncStageOut/TransferDaemon.py

```diff
@@ -79,6 +79,17 @@ def __init__(self, config):
                 else:
                     self.logger.error('Unknown error in mkdir' % e.errno)
                     raise
+
+       if not os.path.isdir("/tmp/DashboardReport"):
```

Comment:
Not a fan of hardcodinf "/tmp", there are python api to create temporary files/directories. Anyhow, I see it is used in other part of the code, so if we want to fix this we should do it in a separate issue.